### ISSUE TITLE
"Recent Commit Messages" is empty

### ISSUE THREAD

**Issue by Turbo87** [ForkIssues/Tracker#394]
I've updated to Fork 1.0.69.4 lately and for some reason the "Recent Commit Messages" list is empty even though I just used Fork to create a few commits 🤔

**Comment by workmanw** [comment]
I reported this issue via email (before I discovered this git repo). And we discovered this happens if you have a branch name that is the same as a file in the root repo. This is because to populate the list of recent commit messages they're using:

```bash
git log %activebranch% --author='%globalauthor%' -n 5 HEAD --pretty=format:%s'
```

The maintainer told me they were going to issue a fix in the next release.

**Comment by DanPristupov** [comment]
Yes, I have the fix planned for 1.0.70.

**Comment by DanPristupov** [comment]
Could you please check if this bug is fixed in 1.0.70?

**Comment by Turbo87** [comment]
hasn't happened in a while, even with 1.0.69 though. I'm assuming that this is fixed, and if it does happen again, I'll reopen the issue. thanks :)
